app/assets/stylesheets/common.css.scss

```diff
@@ -34,7 +34,7 @@
 
 .btn-purple {
   color: white;
-  background-color: $purple;
+  background-color: rgba(65,50,107, .9);
```

Comment:
If that rbg is the same as `$purple` you can use [sass's transparentize](http://sass-lang.com/documentation/Sass/Script/Functions.html#transparentize-instance_method) so we can keep using the color variable for easy reference/editing.